source/content/guides/drupal/drupal-hosted-createdashboard-set8/13-troubleshooting.md

```diff
@@ -1,19 +1,19 @@
 ---
```

Comment:
@stovak , still contains multiple references to D9.  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-hosted-createdashboard-set8/troubleshooting

Reply:
Same